### PR TITLE
JavaScript validation form

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
This is a javascript validatiom form
In this project I have:

- Fix linters errors
- Update my README file
- Executed the validation form
- Added a live demo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -439,6 +447,66 @@ <h3 class="heading-tertiary">Contact me</h3>
...
+            ></textarea>
+          </li>
+          <li>
+            <p class="show-msg" style="display: block;">Please make sure the email is all lower case</p>
+            <input id="submit" type="submit" value="Get in touch" class="button btn-contact">
+          </li>
+        </ul>
+      </form>
+    </section> -->
+
```
Comment (carlos-ssh) [inline index.html:509]: - Kindly remove commented lines

File: style.css [hunk 2]
```diff
@@ -467,6 +469,33 @@ body {
...
+    visibility: hidden;
+    background-color: #fff;
+  }
+
+  to {
+    border-radius: 1rem;
+  }
+}
+
+/* alert message end */
```
Comment (carlos-ssh) [inline style.css:497]: - Remove commented lines

File: script.js [hunk 3]
```diff
@@ -134,3 +134,16 @@ popupOverlay.addEventListener('click', closePopUp);
 btnClosePop.addEventListener('click', closePopUp);
 
 // Popup end /////////////////////
```
Comment (carlos-ssh) [inline script.js:136]: - remove commented lines.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Your project, it's really nice!... I love it.
Kindly remove your commented code, and commented messages.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 3 above)
Comment (Isabellanamutenda) [reply script.js:136]: Resolved

File: index.html (hunk 1 above)
Comment (Isabellanamutenda) [reply index.html:509]: Resolved

File: style.css (hunk 2 above)
Comment (Isabellanamutenda) [reply style.css:497]: Resolved

**Review by bellom — APPROVED** [review]
## Hi @Isabellanamutenda  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**